src/Validation/Validation.php

```diff
@@ -687,7 +687,7 @@ public static function localizedTime($check, string $type = 'datetime', $format
         }
         $method = $methods[$type];
 
-        return Time::$method($check, $format) !== null;
+        return FrozenTime::$method($check, $format) !== null;
```

Comment:
This could be a breaking change depending on how userland code is using it.

Reply:
How so? The method returns a boolean not a Time object.

Reply:
Derp. Reading is hard.